tsconfig.base.json

```diff
@@ -0,0 +1,17 @@
+{
+  "description": "The base configuration for all packages.",
+  "extends": "@vercel/style-guide/typescript",
+  "compilerOptions": {
+    "lib": ["ES2021"],
+    "module": "commonjs",
+    "moduleResolution": "node",
+    "noEmitOnError": true,
+    "noImplicitReturns": true,
```

Comment:
How about adding `noFallthroughCasesInSwitch: true` since most of the others used that too?

Reply:
It's already in the style guide base config: https://github.com/vercel/style-guide/blob/canary/typescript.json#L6. :)